### PR TITLE
Remove feedback for local development

### PR DESCRIPTION
**Pull request by anxolin**
Remove the annoying feedback for local development

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     -->
 
   <title>Cow Swap</title>
-  <script async src="https://w.appzi.io/w.js?token=5ju0G"></script>
+  <!-- <script async src="https://w.appzi.io/w.js?token=5ju0G"></script> -->
```
Comment (alfetopito) [inline public/index.html]: hmmmm, you not removing it only on dev...

Comment (anxolin) [reply public/index.html]: too fast, i didnt commit last thingy

**Review by alfetopito — APPROVED** [review]

**Comment by biocom** [comment]
As mentioned in the chat, for QA purposes it might be good to leave it in place. @alfetopito suggested a toggle to turn it on/off (because of API quotes on the service?). The reason I'd like to see it even in `dev` is because it allows us to test the real UI (e.g. it might happen that the feedback button interferes with any other elements on page).

**Comment by anxolin** [comment]
> As mentioned in the chat, for QA purposes it might be good to leave it in place. @alfetopito suggested a toggle to turn it on/off (because of API quotes on the service?). The reason I'd like to see it even in dev is because it allows us to test the real UI (e.g. it might happen that the feedback button interferes with any other elements on page).

this PR adds it for PRODUCTION, all PRs use this environment for compiling, so we will still have feedback for that.
It just removes it for development, so we don't burn our free tier of the service, and the app loads faster (it loads in each hot-reload)

**Pull request merged**